examples/floating_interdomain/usecases/FloatingVl3_autoscale/README.md

```diff
@@ -0,0 +1,244 @@
+# NSM over interdomain autoscaled vL3 network
+
+## Description
+
+This example shows how to configure autoscaled vL3-network for interdomain.
+
+The diagram is presented below.
+
+![NSM kernel2kernel Diagram](./floating_vl3_autoscale.svg "NSM Kernel2Kernel Scheme")
+
+## Requires
+
+Make sure that you have completed steps from [interdomain](../../)
+
+## Run
+
+**1. Deploy**
+
+1.1. Switch context to the *floating domain*.
+
+```bash
+export KUBECONFIG=$KUBECONFIG3
+```
+
+1.2. Start **vl3 ipam** and register **vl3 network service** in the *floating domain*.
+
+
+Note: *By default we're using ipam prefix is `169.254.0.0/16` and client prefix len is `24`. We also have two vl3 nses in this example. So we are expect to have a two vl3 addresses: `169.254.0.0` and `169.254.1.0` that should be accessible by each client.*
+
+
+```bash
+kubectl apply -k https://github.com/networkservicemesh/deployments-k8s/examples/floating_interdomain/usecases/FloatingVl3_autoscale/cluster3?ref=3175354ab685daf0b2fc5ed7b0b6673b266956d8
+```
+
+1.3. Switch context to the *cluster1*.
+
+```bash
+export KUBECONFIG=$KUBECONFIG1
+```
+
+1.4. First we need to prepare a pod-template for the supplier with a correct **vl3 ipam URL**. Let's use `kustomize` to create it:
+
+```bash
+cat > kustomization.yaml <<EOF
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- https://raw.githubusercontent.com/networkservicemesh/deployments-k8s/3175354ab685daf0b2fc5ed7b0b6673b266956d8/examples/floating_interdomain/usecases/FloatingVl3_autoscale/cluster1/pod-template-base.yaml
+
+patchesStrategicMerge:
+- patch-pod-template.yaml
+EOF
+```
+
+```bash
+cat > patch-pod-template.yaml <<EOF
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nse-vl3-vpp-1
+spec:
+  containers:
+    - name: nse
+      env:
+        - name: NSM_PREFIX_SERVER_URL
+          value: "tcp://vl3-ipam.nsm-system.my.cluster3:5006"
+EOF
+```
+
+Save `pod-template.yaml`:
+```bash
+kubectl kustomize -o 'pod-template.yaml'
+```
```

Comment:
This is looking as a complicaiton. 


Could you please explain why do we need to use this?

Reply:
You are right. 
Fixed